ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -527,7 +529,10 @@ def test_paddle_exp(
         available_dtypes=helpers.get_dtypes("float"),
     ),
 )
+
+
 def test_paddle_cos(
+
     dtype_and_x,
```

Comment:
Some of the blank lines you have added are not needed. Please have a look at the lint errors that appear in the CI checks to correct them.